### PR TITLE
Reflect few contributions on `contribution.md` that were not reflected on #8294

### PR DESCRIPTION
**Pull request by mreraser**
# What does this PR do?

Hello!😃 

As mentioned in the previous [PR](https://github.com/huggingface/diffusers/pull/8294), this PR includes the incorporation of unaddressed contributions to `evaluation.md` and `contribution.md`. Upon review, I found that all contributions were already applied to `evaluation.md`. For the `contribution.md` file, I made corrections to the numbering and added a TOC as instructed in the last PR.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@heuristicwave 

Thank you always for reviewing! 🤗

Best regards,
Jiwook Han

### CODE REVIEW

File: docs/source/ko/conceptual/contribution.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Diffusers 라이브러리와 관련된 모든 질문이나 의견은 [토론 포
 - 이슈가 최신 Diffusers 버전으로 업데이트하면 해결될 수 있는지 확인해주세요. 이슈를 게시하기 전에 `python -c "import diffusers; print(diffusers.__version__)"` 명령을 실행하여 현재 사용 중인 Diffusers 버전이 최신 버전과 일치하거나 더 높은지 확인해주세요.
 - 새로운 이슈를 열 때 투자하는 노력이 많을수록 답변의 품질이 높아지고 Diffusers 이슈 전체의 품질도 향상됩니다.
 
-#### 2.1 재현가능하고 최소한인 버그 리포트
+#### 2.1 재현가능하고 최소한인 버그 리포트 [[21-reproducible-minimal-bug-reports]]
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:84]:
```suggestion
#### 2.1 재현 가능한 최소한의 버그 리포트 [[21-reproducible-minimal-bug-reports]]
```

File: docs/source/ko/conceptual/contribution.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ Diffusers 라이브러리와 관련된 모든 질문이나 의견은 [토론 포
...
 
 새로운 이슈는 일반적으로 다음과 같은 내용을 포함합니다.
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:86]:
```suggestion
```

Comment (heuristicwave) [reply docs/source/ko/conceptual/contribution.md:86]: 최근 문서에는 해당 문장이 삭제된 것 같습니다.

File: docs/source/ko/conceptual/contribution.md [hunk 3]
```diff
@@ -100,12 +99,12 @@ Diffusers 라이브러리와 관련된 모든 질문이나 의견은 [토론 포
 버그 보고서를 열려면 [여기](https://github.com/huggingface/diffusers/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml)를 클릭하세요.
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:99]:
```suggestion
버그 리포트를 열려면 [여기](https://github.com/huggingface/diffusers/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml)를 클릭하세요.
```

File: docs/source/ko/conceptual/contribution.md [hunk 4]
```diff
@@ -81,9 +81,8 @@ Diffusers 라이브러리와 관련된 모든 질문이나 의견은 [토론 포
 - 이슈가 최신 Diffusers 버전으로 업데이트하면 해결될 수 있는지 확인해주세요. 이슈를 게시하기 전에 `python -c "import diffusers; print(diffusers.__version__)"` 명령을 실행하여 현재 사용 중인 Diffusers 버전이 최신 버전과 일치하거나 더 높은지 확인해주세요.
 - 새로운 이슈를 열 때 투자하는 노력이 많을수록 답변의 품질이 높아지고 Diffusers 이슈 전체의 품질도 향상됩니다.
 
-#### 2.1 재현가능하고 최소한인 버그 리포트
+#### 2.1 재현 가능한 최소한의 버그 리포트 [[21-reproducible-minimal-bug-reports]]
 
-새로운 이슈는 일반적으로 다음과 같은 내용을 포함합니다.
 
 버그 보고서는 항상 재현 가능한 코드 조각을 포함하고 가능한 한 최소한이어야 하며 간결해야 합니다.
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:87]:
```suggestion
버그 리포트는 항상 재현 가능한 코드 조각을 포함하고 가능한 한 최소한이어야 하며 간결해야 합니다.
```

File: docs/source/ko/conceptual/contribution.md [hunk 5]
```diff
@@ -268,7 +267,7 @@ Diffusers 예제는 [examples](https://github.com/huggingface/diffusers/tree/mai
 연구용 훈련 예제는 [examples/research_projects](https://github.com/huggingface/diffusers/tree/main/examples/research_projects)에 위치하며, 공식 훈련 예제는 `research_projects` 및 `community` 폴더를 제외한 [examples](https://github.com/huggingface/diffusers/tree/main/examples)의 모든 폴더를 포함합니다.
 공식 훈련 예제는 Diffusers의 핵심 메인테이너가 유지 관리하며, 연구용 훈련 예제는 커뮤니티가 유지 관리합니다.
 이는 공식 파이프라인 vs 커뮤니티 파이프라인에 대한 [6. 커뮤니티 파이프라인 기여하기](#6-contribute-a-community-pipeline)에서 제시한 이유와 동일합니다: 핵심 메인테이너가 diffusion 모델의 모든 가능한 훈련 방법을 유지 관리하는 것은 현실적으로 불가능합니다.
-Diffusers 핵심 메인테잉너와 커뮤니티가 특정 훈련 패러다임을 너무 실험적이거나 충분히 인기 없는 것으로 간주하는 경우, 해당 훈련 코드는 `research_projects` 폴더에 넣고 작성자가 유지 관리해야 합니다.
+Diffusers 핵심 메인테이너와 커뮤니티가 특정 훈련 패러다임을 너무 실험적이거나 충분히 인기 없는 것으로 간주하는 경우, 해당 훈련 코드는 `research_projects` 폴더에 넣고 작성자가 유지 관리해야 합니다.
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:270]:
```suggestion
Diffusers 핵심 메인테이너와 커뮤니티가 특정 훈련 패러다임을 너무 실험적이거나 충분히 대중적이지 않다고 판단한다면, 해당 훈련 코드는 `research_projects` 폴더에 넣고 작성자에 의해 관리되어야 합니다.
```

File: docs/source/ko/conceptual/contribution.md [hunk 6]
```diff
@@ -323,7 +322,7 @@ PR에 원본 코드베이스/논문 링크를 추가하고, 가능하면 PR에
 
 PR에서 막힌 경우나 도움이 필요한 경우, 첫 번째 리뷰나 도움을 요청하는 메시지를 남기는 것을 주저하지 마세요.
 
-#### Copied from mechanism
+#### Copied from mechanism [[copied-from-mechanism]]
 
 `# Copied from mechanism` 은 파이프라인, 모델 또는 스케줄러 코드를 추가할 때 이해해야 할 독특하고 중요한 기능입니다. Diffusers 코드베이스 전체에서 이를 자주 볼 수 있는데, 이를 사용하는 이유는 코드베이스를 이해하기 쉽고 유지 관리하기 쉽게 유지하기 위함입니다. `# Copied from mechanism` 으로 표시된 코드는 복사한 코드와 정확히 동일하도록 강제됩니다. 이를 통해 `make fix-copies`를 실행할 때 많은 파일에 걸쳐 변경 사항을 쉽게 업데이트하고 전파할 수 있습니다.
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:327]:
```suggestion
`# Copied from mechanism` 은 파이프라인, 모델 또는 스케줄러 코드를 추가할 때 이해해야 할 독특하고 중요한 기능입니다. 이것은 Diffusers 코드베이스 전반에서 볼 수 있으며, 이를 사용하는 이유는 코드베이스를 이해하고 유지 관리하기 쉽게 만들기 위해서입니다. `# Copied from mechanism` 으로 표시된 코드는 복사한 코드와 정확히 동일하도록 강제됩니다. 이렇게 하면 `make fix-copies`를 실행할 때마다 여러 파일에 걸쳐 변경 사항을 쉽게 업데이트하고 전파할 수 있습니다.
```

File: docs/source/ko/conceptual/contribution.md [hunk 7]
```diff
@@ -359,7 +358,7 @@ class AltDiffusionPipelineOutput(BaseOutput):
 6. 포맷팅. 파이썬 코드 구문으로 코드를 포맷팅하고, 일반 코드 구문으로 에러 메시지를 포맷팅해주세요. 자세한 내용은 [공식 GitHub 포맷팅 문서](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)를 참조하세요.
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:358]:
```suggestion
6. 포맷팅. 코드를 파이썬 코드 구문으로, 에러 메시지를 일반 코드 구문으로 형식화하여 이슈를 깔끔하게 작성하세요. 자세한 내용은 [GitHub 공식 포맷팅 문서](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)를 참조하세요.
```

File: docs/source/ko/conceptual/contribution.md [hunk 8]
```diff
@@ -359,7 +358,7 @@ class AltDiffusionPipelineOutput(BaseOutput):
...
 7. 이슈를 해결해야 하는 티켓이 아니라, 잘 작성된 백과사전 항목으로 생각해보세요. 추가된 이슈는 공개적으로 사용 가능한 지식에 기여하는 것입니다. 잘 작성된 이슈를 추가함으로써 메인테이너가 문제를 해결하는 데 도움을 주는 것뿐만 아니라, 전체 커뮤니티가 라이브러리의 특정 측면을 더 잘 이해할 수 있도록 도움을 주는 것입니다.
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:359]:
```suggestion
7. 여러분의 이슈를 단순히 해결해야 할 티켓으로 생각하지 말고, 잘 작성된 백과사전 항목으로 생각해보세요. 추가된 모든 이슈는 공개적으로 이용 가능한 지식에 대한 기여입니다. 잘 작성된 이슈를 추가함으로써 메인테이너가 여러분의 이슈를 더 쉽게 해결할 수 있게 할 뿐만 아니라, 전체 커뮤니티가 라이브러리의 특정 측면을 더 잘 이해할 수 있도록 도움을 주게 됩니다.
```

File: docs/source/ko/conceptual/contribution.md [hunk 9]
```diff
@@ -359,7 +358,7 @@ class AltDiffusionPipelineOutput(BaseOutput):
...
 
-## 좋은 PR 작성 방법
+## 좋은 PR 작성 방법 [[how-to-write-a-good-pr]]
 
 1. 카멜레온이 되세요. 기존의 디자인 패턴과 구문을 이해하고, 코드 추가가 기존 코드베이스에 매끄럽게 흐르도록 해야 합니다. 기존 디자인 패턴이나 사용자 인터페이스와 크게 다른 PR은 병합되지 않습니다.
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:363]:
```suggestion
1. 카멜레온이 되세요. 기존의 디자인 패턴과 구문을 이해하고, 여러분이 추가하는 코드가 기존 코드베이스와 자연스럽게 어우러지도록 해야 합니다. 기존 디자인 패턴이나 사용자 인터페이스와 크게 다른 PR은 병합되지 않습니다.
```

File: docs/source/ko/conceptual/contribution.md [hunk 10]
```diff
@@ -359,7 +358,7 @@ class AltDiffusionPipelineOutput(BaseOutput):
...
 2. 초점을 맞추세요. 하나의 문제만 해결하는 PR을 작성해야 합니다. "추가하면서 다른 문제도 해결하기"에 빠지지 않도록 주의하세요. 여러 개의 관련 없는 문제를 해결하는 PR을 작성하는 것은 리뷰하기가 훨씬 어렵습니다.
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:364]:
```suggestion
2. 레이저처럼 집중하세요. Pull Request는 하나의 문제, 오직 하나의 문제만 해결해야 합니다. "이왕 추가하는 김에 다른 문제도 고치자"는 함정에 빠지지 않도록 주의하세요. 여러 개의 관련 없는 문제를 해결하는 한 번에 해결하는 Pull Request들은 검토하기가 훨씬 더 어렵습니다.
```
앞선, 카멜레온 비유를 따라 레이저 은유법을 적용하고, solve one problem and one problem only 느낌을 살려 변경해보았습니다.

Comment (heuristicwave) [reply docs/source/ko/conceptual/contribution.md:364]: 아래 PR들도 형식을 맞추는것이 좋을 것 같습니다.

File: docs/source/ko/conceptual/contribution.md [hunk 11]
```diff
@@ -494,7 +493,7 @@ $ python -m unittest discover -s tests -t . -v
 $ python -m unittest discover -s examples -t examples -v
 ```
 
-### upstream(main)과 forked main 동기화하기
+### upstream(main)과 forked main 동기화하기 [[syncing-forked-main-with-upstream-huggingface-main]]
```
Comment (heuristicwave) [inline docs/source/ko/conceptual/contribution.md:496]:
```suggestion
### upstream(HuggingFace) main과 forked main 동기화하기 [[syncing-forked-main-with-upstream-huggingface-main]]
```

**Review by heuristicwave — COMMENTED** [review]
The document was very long, so the review took a long time. Thanks to this, I was able to read the contribute document again and learn what I should take to heart.

**Comment by mreraser** [comment]
> The document was very long, so the review took a long time. Thanks to this, I was able to read the contribute document again and learn what I should take to heart.

Thank you so much for your continuous and dedicated reviews. Your feedback has been invaluable, and I truly appreciate the time and effort you've invested in. 

Have a nice day😊

**Comment by mreraser** [comment]
Hello @stevhliu 😊, may I ask for your final review of the latest changes when you have a moment? If everything looks good, I would greatly appreciate it if you could merge them. Thank you so much! 🙏

**Review by stevhliu — APPROVED** [review]
Thanks for following up and completing! 🤗

**Pull request merged**